src/fr/standards/what_we_heard_report.md

```diff
@@ -4,22 +4,48 @@ layout: layouts/base.njk
 description: "Entre le 31 octobre et le 30 novembre 2022, la Division de la politique numérique et de la performance du Secrétariat du Conseil du Trésor du Canada - Bureau du dirigeant principal de l'information (<abbr>SCT-BDPI</abbr>) a entrepris un engagement ciblé sur la <em>Norme d'accessibilité des technologies de l'information et de la communication (<abbr>TIC</abbr>)</em>. Cette page détaille les principaux thèmes qui ont émergé et les prochaines étapes à suivre en ce qui concerne la norme."
 ---
 
-## Objectif 
+<div class="row">
+
+  <div class="col-md-4 col-md-push-8 mrgn-tp-lg">
+    <a class="gc-dwnld-lnk" href="/docs/sitka-wwhr-fr.pptx">
```

Comment:
Commentaire:
Il semble que le pattern du download link wrap le tout. Il va falloir qu'il soit revisé.

Reply:
I kind of like it like that but it look like the image and the text is a separate link.

Reply:
Yeah but by definition it is not optimal, we can discuss offline about it.